project/bridge.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+import networkx
+import pydot
+import utils
+
+__all__ = ["graph_info", "create_and_export", "shutdown"]
```

Comment:
По заданию у вас должны быть следующие команды. (названия примерные)
```suggestion
command_names = ["get_graph_info", "create_two_cycles_graph", "save_to_dot", "quit"]
```

Reply:
Исправлено.